user-docs/pupil-capture.md

```diff
@@ -257,9 +257,9 @@ Fixation detectors classify fixations using dispersion and duration.
 classify pupil positions as fixations if they are within the dispersion for the
 duration of time specified.
 
-The `Fixation Detector 3D` plugin uses the eye model's 3d orientation angle to
-calculate the dispersion. The settings allow you to set the duration in seconds
-and the maximum dispersion amount in degrees.
+The `Fixation Detector` plugin uses depending on availability the eye model's 3d orientation angle or
```

Comment:
Edit:

The `Fixation Detector` plugin uses the eye model's 3d orientation angle (if available) or 2d gaze pixel position to calculate dispersion. The plugin enable you to set the duration in seconds and the maximum dispersion in degrees for 3d data and pixels for 2d data.